core/src/main/java/hudson/DNSMultiCast.java

```diff
@@ -77,16 +84,102 @@ public Object call() {
 
     public void close() {
         if (jmdns!=null) {
-//            try {
+            try {
                 jmdns.abort();
                 jmdns = null;
-//            } catch (final IOException e) {
-//                LOGGER.log(Level.WARNING,"Failed to close down JmDNS instance!",e);
-//            }
+            } catch (final IOException e) {
+                LOGGER.log(Level.WARNING,"Failed to close down JmDNS instance!",e);
+            }
         }
     }
 
     private static final Logger LOGGER = Logger.getLogger(DNSMultiCast.class.getName());
 
-    public static boolean disabled = SystemProperties.getBoolean(DNSMultiCast.class.getName()+".disabled");
+    public static boolean disabled = SystemProperties.getBoolean(DNSMultiCast.class.getName()+".disabled", true);
```

Comment:
I might be wrong but this feature does not seem widely used so it can be disabled by default. If it is approved, I'll change the documentation.

Reply:
I would suggest doing it outside this PR. Let's create a follow-up ticket and start a dev list discussion